src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
addEventListener is technically from dom level 2 events :)  HTML4 started out with what are known as DOM level 0 events only (onclick="", etc.). So I'd probably call that dom2.